hooks/useLocal.tsx

```diff
@@ -0,0 +1,161 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
```

Comment:
`hooks/useLocal.tsx` should be `hooks/useLocal.ts` since it doesn't contain JSX. This is a common React Native convention.

Reply:
Look for other places where this can be updated.